test_classes_list.txt

```diff
@@ -41,4 +41,5 @@ ipv8/test/dht/test_routing.py:TestBucket
 ipv8/test/dht/test_routing.py:TestRoutingTable
 ipv8/test/dht/test_storage.py:TestStorage
 
-ipv8/test/REST/test_attestation_endpoint.py:TestAttestationEndpoint
\ No newline at end of file
+ipv8/test/REST/attestationendpoint/test_attestation_endpoint.py:TestAttestationEndpoint
+ipv8/test/REST/dht/test_dht_endpoint.py:TestDHTEndpoint
```

Comment:
We need a blank line at the end of this file, otherwise one of the scripts dies.

Reply:
Got it! Will fix that soon. Other than this, are there any other issues with the code, or the idea behind the implementation?

Reply:
I'd have to do an in-depth review. Are you done changing stuff?